solvers/qpSpline/Polynomial.cpp

```diff
@@ -0,0 +1,131 @@
+#include "Polynomial.h"
+#include <cassert>
+#include <algorithm>
+
+using namespace std;
+using namespace Eigen;
+
+Polynomial::Polynomial(Eigen::Ref<Eigen::VectorXd> const& coefficients) :
```

Comment:
sure.  that's fine.

On Apr 10, 2015, at 8:36 AM, Twan Koolen <notifications@github.com> wrote:

> In solvers/qpSpline/Polynomial.cpp:
> 
> > @@ -0,0 +1,131 @@
> > +#include "Polynomial.h"
> > +#include <cassert>
> > +#include <algorithm>
> > +
> > +using namespace std;
> > +using namespace Eigen;
> > +
> > +Polynomial::Polynomial(Eigen::Ref<Eigen::VectorXd> const& coefficients) :
> but maybe in a different subdirectory?
> 
> —
> Reply to this email directly or view it on GitHub.
>